lib/optimizer/level-1/optimize.js

```diff
@@ -89,16 +89,25 @@ function optimizeBorderRadius(property) {
   }
 }
 
+/**
+ * @param {string} name
+ * @param {string} value
+ * @param {Object} compatibility
+ * @return {string}
+ */
 function optimizeColors(name, value, compatibility) {
-  if (value.indexOf('#') === -1 && value.indexOf('rgb') == -1 && value.indexOf('hsl') == -1) {
+  if (!value.match(/#|rgb|hsl/gi)) {
```

Comment:
I wonder if this change may lead to a slight performance degradation, but let's merge it and fix if needed.